Simplists/Sources/Views/List/ListView.swift

```diff
@@ -175,6 +175,7 @@ struct ListView: View {
                                                     Text("Delete all items")
                                                     Image(systemName: "circle.dashed")
                                                 }
+                                                .foregroundColor(Color.red)
```

Comment:
This was a mistake to leave in. The `.foregroundColor` has no effect on the menu item's color. Will remove this line and line 187 below (deletions to be included in some future release).